docs/prerequisites.rst

```diff
@@ -100,7 +100,7 @@ Install MongoDB server using YUM: ::
 
 Enable the Node.js YUM repository: ::
 
-    curl -sL https://rpm.nodesource.com/setup | sudo bash -
+    curl -sL https://rpm.nodesource.com/setup_4.x | sudo bash
```

Comment:
Is there a good reason to remove the `-`? Both ways should work, but these instructions are now inconsistent with the APT repository instructions, as well as the [node.js instructions](https://nodejs.org/en/download/package-manager/#enterprise-linux-and-fedora).